services/fileServices/MdPageServices/CollectionPageService.js

```diff
@@ -13,7 +13,7 @@ class CollectionPageService {
   }
 
   async create(reqDetails, { fileName, collectionName, content, frontMatter }) {
-    if (titleSpecialCharCheck(fileName))
+    if (titleSpecialCharCheck(fileName, true))
```

Comment:
nit:
Would it be a good idea to modify the function to take in an object with destructured arguments, e.g. `titleSpecialCharCheck({fileName, isFile: true})`? may be hard to look back and figure out what `true` refers to?